db/migrate/20200526015005_encrypt_password_in_audit.rb

```diff
@@ -0,0 +1,29 @@
+class EncryptPasswordInAudit < ActiveRecord::Migration[6.0]
+  def encrypt_upstream_password_for_root_repository_audits
+    ::Katello::RootRepository.all.each do |repository|
+      repository.audits.each do |audit|
+        if audit.audited_changes.key? "upstream_password"
+          changes = audit.audited_changes
+          name = :upstream_password.to_s
+          change = changes[name]
+
+          if change.is_a? Array
+            changes[name] = change.map { |_| AuditExtensions::REDACTED }
+          elsif change
+            changes[name] = AuditExtensions::REDACTED
+          end
```

Comment:
Could refactor this logic a bit ? may be in the root repo add a `update_password_for_audit(audit)` method and call it here.